src/fuzz_introspector/datatypes/fuzzer_profile.py

```diff
@@ -438,6 +438,11 @@ def _load_coverage(self, target_folder: str) -> None:
                     self.all_class_functions
                 )
         elif self.target_lang == "jvm":
+            self.coverage = code_coverage.load_llvm_coverage(
```

Comment:
This change looks a bit strange to met: if the target is `jvm` then use `load_llvm_coverage`? LLVM coverage is only for C/C++, or is it also for java?

Reply:
Because we does not have coverage report yet for java, so this is a temporary measure to use the logic of the cpp one to build up an empty coverage report to show.